sdk/cognitivelanguage/ai-language-conversations/src/conversationAnalysisClient.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+import { bearerTokenAuthenticationPolicy } from "@azure/core-rest-pipeline";
+import { PollerLike, PollOperationState } from "@azure/core-lro";
+import {
+  ConversationAnalysisClientOptionalParams,
+  AnalyzeConversationTaskUnion,
+  AnalyzeConversationOptionalParams,
+  AnalyzeConversationResponse,
+  AnalyzeConversationJobsInput,
+  ConversationAnalysisOptionalParams,
+  ConversationAnalysisResponse
+} from "./generated/models";
+import { conversationAnalysisAzureKeyCredentialPolicy } from "./azureKeyCredentialPolicy";
+import { DEFAULT_COGNITIVE_SCOPE, SDK_VERSION } from "./constants";
+import { isTokenCredential, KeyCredential, TokenCredential } from "@azure/core-auth";
+import { ConversationAnalysisClient as GeneratedClient } from "./generated";
+import { createTracingClient, TracingClient } from "@azure/core-tracing";
+
+/** @internal */
+export class ConversationAnalysisClient {
+  private readonly _client: GeneratedClient;
+  private readonly _tracing: TracingClient;
+
+  /**
+   * Initializes a new instance of the ConversationAnalysisClient class.
+   * @param endpoint Supported Cognitive Services endpoint (e.g.,
+   *                 https://<resource-name>.api.cognitiveservices.azure.com).
+   * @param options The parameter options
+   */
+   constructor(
+    endpoint: string,
+    credential: TokenCredential | KeyCredential,
+    options: ConversationAnalysisClientOptionalParams = {}
+  ) {
+    if (endpoint === undefined) {
+      throw new Error("'endpoint' cannot be null");
+    }
+
+    // Initializing default values for options
+    if (!options) {
+      options = {};
+    }
+    const defaults: ConversationAnalysisClientOptionalParams = {
+      requestContentType: "application/json; charset=utf-8"
+    };
+
+    const packageDetails = `azsdk-js-ai-language-conversations/1.0.0-beta.1`;
+    const userAgentPrefix =
+      options.userAgentOptions && options.userAgentOptions.userAgentPrefix
+        ? `${options.userAgentOptions.userAgentPrefix} ${packageDetails}`
+        : `${packageDetails}`;
+
+    const optionsWithDefaults = {
+      ...defaults,
+      ...options,
+      userAgentOptions: {
+        userAgentPrefix
+      },
+      baseUri: options.endpoint ?? options.baseUri ?? "{Endpoint}/language"
+    };
```

Comment:
This logic is already there in the generated client, could you update this bit similar to what the constructor for @azure/ai-language-text does.